### ISSUE TITLE
Potential bug in 2 image augmentation code for base step training

### ISSUE THREAD

**Issue by prachigarg23** [CanPeng123/FSCIL_ALICE#7]
Hi @CanPeng123 

In train.py > train_two_image() :
Line 159 `angular_loss = 0.5 * angular_loss_1 + 0.5 + angular_loss_2`

Is the + 0.5 a bug? In the paper it says that loss for 2 augmentation images is averaged before back propagation.

**Comment by CanPeng123** [comment]
Hi, 
Yes, it is a typo mistake when I cleaned and uploaded the code. 
I have changed it. Sorry for the inconvenience.
